From 786f8efde24951c73f3c022d0f96733b78d695ae Mon Sep 17 00:00:00 2001
From: dervoeti <lukas.krug@stackable.tech>
Date: Fri, 13 Jun 2025 15:38:45 +0200
Subject: Build hadoop-client-modules before hadoop-dist

This is needed, because dist depends on parts of client-modules. At least when specifying a custom version when building Hadoop, Maven for some reason does not build the client-modules before dist and the build fails.
---
 pom.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pom.xml b/pom.xml
index de001775abaf39477f94e133ce9be2399bbc36f2..ccb15235c8407db1e914bed2443259b0fc3a3cb4 100644
--- a/pom.xml
+++ b/pom.xml
@@ -133,9 +133,9 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/x
     <module>hadoop-yarn-project</module>
     <module>hadoop-mapreduce-project</module>
     <module>hadoop-tools</module>
+    <module>hadoop-client-modules</module>
     <module>hadoop-dist</module>
     <module>hadoop-minicluster</module>
-    <module>hadoop-client-modules</module>
     <module>hadoop-build-tools</module>
     <module>hadoop-cloud-storage-project</module>
   </modules>
